src/state/toastMessages/actions.ts

```diff
@@ -0,0 +1,38 @@
+import { v4 as uuidv4 } from 'uuid';
+
+import { Toast } from 'app/consts';
+
+export enum ToastMessageAction {
+  AddToastMessage = 'AddToastMessage',
+  HideToastMessage = 'HideToastMessage',
+}
+
+export interface AddToastMessageAction {
+  type: ToastMessageAction.AddToastMessage;
+  payload: Toast;
+}
+
+export interface HideToastMessageAction {
+  type: ToastMessageAction.HideToastMessage;
+  payload: Toast;
+}
+
+export type ToastMessageActionType = AddToastMessageAction | HideToastMessageAction;
+
+export const addToastMessage = ({
+  title,
+  description,
+  milisecondsAfterHide = 5500,
+}: {
+  title: string;
+  description: string;
+  milisecondsAfterHide?: number;
```

Comment:
Maybe interface? Or use the existing one with Omit<>?

Reply:
added